### DISCUSSION TITLE
Rendering a Document

### DISCUSSION THREAD

**Discussion by unkn00wn113** [quarto-dev/quarto-cli#8502] [Q&A] [answered]
### Description

Hi, I am new to Quarto and I am trying to render a document on Jupyternotebook on VS and I have Quarto installed in the extensions however when I run this code: quarto render united1_quarto.ipynb –to pdf it gives me an error stating: quarto : The term 'quarto' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1

**Comment by mcanouil** [comment]
Did you install Quarto CLI?
See <https://quarto.org/docs/get-started/>.
I suggest you go through the get started guide which will help get started as a new Quarto user.

**Reply by unkn00wn113** [reply]
Hi, I did and it randomly started working and now every time I try to render it, it gives me an error: 
pandoc: to: withBinaryFile: does not exist (No such file or directory)

**Reply by mcanouil** [reply]
What's your environment?
What if you type `quarto check` in a terminal?

**Reply by unkn00wn113** [reply]
it is being very weird, for some reason one I ran it, it output a latex file, and now it just give me that error and giving me png files of my graphs? 
and I am running it on .conda
and if I type quarto check it gives me:
Quarto 1.4.549
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.4.549
      Path: C:\Users\user\AppData\Local\Programs\Quarto\bin
      CodePage: unknown

[>] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Tex:  (not detected)

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

      Version: 4.2.2
      Path: C:/PROGRA~1/R/R-42~1.2
      LibPaths:
        - C:/Users/user/AppData/Local/R/win-library/4.2
        - C:/Program Files/R/R-4.2.2/library
      knitr: 1.42
      rmarkdown: 2.20

[>] Checking Knitr engine render......OK

**Reply by mcanouil** [reply]
You need to provide a fully reproducible example, possibly via a Git repository.
Also, please format your post using code blocks to share code or terminal outputs.

From the output of Quarto, it does not seem you have any LaTeX distribution, so there is no way Quarto can render to PDF, without the tools to do so.
See <https://quarto.org/docs/output-formats/pdf-engine.html>.

**Reply by unkn00wn113** [reply]
Hi,

Sorry about that, I am not sure how to do that as I am not familiar with this as it is my first time.

And will use code blocks. However, I was able to install LaTeX distributions however I am still getting the same error:
```
(stats) C:\Users\user\Desktop\stat322\.conda>quarto render untitled.ipynb –to pdf
pandoc –to pdf
  to: html
  output-file: untitled.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  toc: true

metadata
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  title: 'Tips for Python, JupyterLab, and Quarto'
  author: user
  date: 01/11/2024

pandoc: to: withBinaryFile: does not exist (No such file or directory)
```

**Reply by mcanouil** [reply, marked as answer]
I believe you are missing a `-` in your command, i.e., `--to`.
